subprojects/distributions/src/changes/changes-since-4.0.txt

```diff
@@ -0,0 +1,5 @@
+Implemented interface org.gradle.api.initialization.IncludedBuild;@Incubating interface has been removed
+Method org.gradle.api.initialization.IncludedBuild;@Incubating interface has been removed
+Class org.gradle.api.invocation.Gradle;All regressions accepted
```

Comment:
Let's phrase it different: If we break `Gradle` again by changing another method, then the build will not break since we accepted all the changes to `Gradle` already. This is what I would like to avoid and why I added the complete "change" of `Gradle` to accepted changes. Does this make sense to you?